static/app/components/forms/controls/selectOption.tsx

```diff
@@ -43,6 +43,8 @@ function SelectOption(props: Props) {
         <MenuListItem
           {...itemProps}
           {...innerProps}
+          role={isMultiple ? 'menuitemcheckbox' : 'menuitemradio'}
+          aria-checked={isSelected}
```

Comment:
`menuitemcheckbox` and `menuitemradio` require this attribute, which will be helpful in tests to understand if the item has been checked or not